Kconfig

```diff
@@ -0,0 +1,8 @@
+# General configuration options
+
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config ARG_BUFFER_SIZE
+         int "Buffer size in bytes"
+	 default 1024
```

Comment:
Please add the `help` section to describe what this is.

Reply:
@dcpleung added the help section. Thanks!!